king_phisher/client/dialogs/configuration.py

```diff
@@ -274,6 +290,22 @@ def interact(self):
 		self.dialog.destroy()
 		return response
 
+	def save_proxy_settings(self):
+		proxy_url = urllib.parse.urlparse(self.gtk_builder_get('entry_proxy_url').get_text())
```

Comment:
Would be a good idea to strip the whitespace from the URL and username at least. Passwords I guess could technically have whitespace so I'll leave that up to you but it would be a bit more forgiving on input from the user in cases where maybe they're pasting the value in from some document.